<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendWizard</class>
 <widget class="QWizard" name="SendWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>247</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preview and Send</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images-symbian.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight|QWizard::NoBackButtonOnLastPage|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="previewPage">
   <property name="title">
    <string>Preview Messages</string>
   </property>
   <property name="subTitle">
    <string>See how it looks.</string>
   </property>
   <attribute name="pageId">
    <string notr="true"/>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Show for:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="previewContactCombo"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="messagePreviewEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="generatePage">
   <property name="title">
    <string>Generate</string>
   </property>
   <property name="subTitle">
    <string>What to do with the messages.</string>
   </property>
   <attribute name="pageId">
    <string notr="true"/>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QRadioButton" name="saveFilesRadio">
      <property name="text">
       <string>Save as files to folder:</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="folderEdit">
      <property name="text">
       <string>C:\MessageMerge</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="sendTextRadio">
      <property name="text">
       <string>Send Text Messages</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>234</width>
        <height>126</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="draftTextRadio">
      <property name="text">
       <string>Save as Draft Text Messages</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="processingPage">
   <property name="title">
    <string>Processing</string>
   </property>
   <property name="subTitle">
    <string>Please wait...</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="processingLabel1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>In your case you would have to set up the connection manually, but you shouldn't use the &quot;on_&quot; prefix for the slot's name because connectSlotsByName is also called at the end of setupUi which would again throw the warning that there is no matching signal for the slot.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>78</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="processingLabel2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="finishPage">
   <property name="title">
    <string>Finished.</string>
   </property>
   <property name="subTitle">
    <string>Processing has completed.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="finishLabel">
      <property name="text">
       <string>MessageMerge has finished processing.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images-symbian.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>refreshMessagePreview()</slot>
 </slots>
</ui>
